examples/set-labels-advanced/fn-config.yaml

```diff
@@ -7,9 +7,3 @@ metadata:
 labels:
   color: orange
   fruit: apple
-additionalLabelFields:
-  - kind: MyResource
-    create: true
-    group: dev.example.com
-    path: spec/selector/labels
-    version: v1
```

Comment:
IMO it doesn't make a lot of sense to remove these lines even if we want to deprecated `additionalLabelFields`.
We can change the example name to something like `set-labels-deprecated`.
If some users are using it, it can be a problem if they can't find an example.

Reply:
Why cannot users find examples? I think the idea is that users find docs and examples from the website, which points to a specific released version. I guess that could be more confusing to keep an unsupported example.